attributes/network/9_account.adoc

```diff
@@ -28,57 +29,90 @@ Some example of dripping wallet's use cases include;
 *   automatic transfer of funds to an execution network as gas payment.
 
 == Requirements
-*   An account MUST be created with an owner.
-    An account without an owner should be considered deactivated and is safe to be removed.
+*   Every account MUST have at least one owner.
+    The server SHOULD return an error if the last owner attempts to remove itself.
 *   Accounts are referred to by their identities.
 *   A server creating an account MUST use a subresource identity of its own identity for the identity of the account, or a public key it owns.
 
 == Detailed Design
-The simplified design for this attribute is to add the basics for account management, and let features (separate attributes) decide how the account act.
-Each type of account features should be additional attributes for the network.
+The simplified design for this attribute is to add the basics for account management, and let account features decide how the account act.
+Each type of account features should be distinct.
+Features
 A server might support multiple account features, and apply multiple compatible features to the same account.
 
 A new namespace is created through this attribute; `account`, which contains the following basic methods implemented by this attribute:
 
 * `account.create`.
-    Creates an account and sets its owner.
+    Creates an account and sets its owner(s).
+* `account.listRoles`
+    List all the roles supported by the account.
+* `account.getRoles`
+    Get the roles of one or multiple identities.
+* `account.addRole`
+    Get the roles of one or multiple identities.
+* `account.removeRole`
+    Get the roles of one or multiple identities.
 * `account.info`.
     Retrieves information related to an account.
-* `account.set`.
-    Sets information related to the account.
+* `account.setName`.
+    Set the name to a new one.
 * `account.delete`.
     Deletes an account.
     The identity may not be reusable.
 
 === Information
 An account has metadata associated with it.
-For example, it can have a name.
+For example, it can have a name, or a list of features and their arguments.
 
-The information is set when creating, and can be retrieved using `account.info`, and updated when using `account.set`.
+Once an account has been created, its list of features is immutable and cannot be changed.
```

Comment:
Adding new features is not a problem since the feature list is fixed at account creation. Removing support for a feature would disable accounts and that's not specified at the moment. Not sure how we could even handle that.